### PR TITLE
fix: clean completed migration job before reinitialize on-prem demo

### PR DESCRIPTION
**Pull request by exu**
## Pull request description 

The idea is to clean long TTL migration jobs when they exist and we're trying to recreate them. 

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: cmd/kubectl-testkube/commands/common/helper.go [hunk 1]
```diff
@@ -162,6 +162,30 @@ func updateHelmRepo(helmPath string, dryRun bool, isOnPrem bool) *CLIError {
...
 
+// It cleans existing migrations job with long TTL
+func CleanExistingCompletedMigrationJobs(namespace string) (cliErr *CLIError) {
+	kubectlPath, cliErr := lookupKubectlPath()
+	if cliErr != nil {
+		return cliErr
+	}
+
+	// Clean the job only when it's found and it's state is successful - ignore pending migrations.
+	succeeded, _ := runKubectlCommand(kubectlPath, []string{"get", "job", "testkube-enterprise-api-migrations", "-n", namespace, "-o", "jsonpath={.status.succeeded}"})
```
Comment (vsukhin) [inline cmd/kubectl-testkube/commands/common/helper.go:173]: is it okay to skip error or it will give us additional info?

**Review by vsukhin — APPROVED** [review]
one question

**Comment by exu** [comment]
> one question

TBH I'm not sure here if I want to inform user about false positives here

File: cmd/kubectl-testkube/commands/common/helper.go (hunk 1 above)
Comment (exu) [reply cmd/kubectl-testkube/commands/common/helper.go:173]: I think I would need to filter errors related to state when there is no migration job at all 
Assuming happy path for now

**Pull request merged**